applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -309,20 +309,9 @@ def FinalizeSolutionStep(self):
 
         if self._TimeBufferIsInitialized():
             # Recompute the distance field according to the new level-set position
-            if (self._reinitialization_type == "variational"):
+            if self._reinitialization_type == "variational" or self._reinitialization_type == "parallel":
                 self._GetDistanceReinitializationProcess().Execute()
-            elif (self._reinitialization_type == "parallel"):
-                layers = self.settings["parallel_redistance_max_layers"].GetInt()
-                max_distance = 1.0 # use this parameter to define the redistancing range
-                # if using CalculateInterfacePreservingDistances(), the initial interface is preserved
-                self._GetDistanceReinitializationProcess().CalculateDistances(
-                    self.main_model_part,
-                    self._levelset_variable,
-                    KratosMultiphysics.NODAL_AREA,
-                    layers,
-                    max_distance,
-                    self._GetDistanceReinitializationProcess().CALCULATE_EXACT_DISTANCES_TO_PLANE) #NOT_CALCULATE_EXACT_DISTANCES_TO_PLANE)
-
+        
             if (self._reinitialization_type != "none"):
                 KratosMultiphysics.Logger.PrintInfo(self.__class__.__name__, "Redistancing process is finished.")
```

Comment:
Very minor, but we can now move the content of this `if`statement to the one above.

Reply:
done!